apps/openmw/mwbase/windowmanager.hpp

```diff
@@ -219,6 +219,7 @@ namespace MWBase
             virtual void unsetSelectedWeapon() = 0;
 
             virtual void showCrosshair(bool show) = 0;
+            virtual void setCrosshairOwned(bool owned) = 0;
```

Comment:
We could do this without interface additions to `WindowManager`. For example, update the stolen state in the WindowManager::setFocusObject function.